administration/src/components/verification/VerificationController.tsx

```diff
@@ -0,0 +1,110 @@
+import { Button, Card, H1, Spinner } from '@blueprintjs/core'
+import React, { useCallback, useContext } from 'react'
+import { useLocation, useParams } from 'react-router-dom'
+import styled from 'styled-components'
+import JsonFieldView from '../applications/JsonFieldView'
+import ErrorHandler from '../../ErrorHandler'
+import { useGetApplicationQuery, useVerifyApplicationVerificationMutation } from '../../generated/graphql'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import NotFoundView from '../NotFoundView'
+import { useAppToaster } from '../AppToaster'
+
+const CARD_PADDING = 20
+
+const ApplicationViewCard = styled(Card)`
+  width: 800px;
+  margin: 10px;
+`
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  justify-content: center;
+  padding: ${CARD_PADDING}px;
```

Comment:
if you just use it once, i think you can just use
```suggestion
  padding: 20px;
```

Reply:
I tried to stay consistent with the already existing layout, and therefore copied the constant. On second thought it makes much more sense to export & import the value than to copy it.